pkg/system-probe/config/adjust_npm.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 const (
-	maxConnsMessageBatchSize     = 1000
-	maxOffsetThreshold           = 3000
-	defaultMaxProcessesTracked   = 1024
-	defaultMaxTrackedConnections = 65536
+	maxConnsMessageBatchSize                                = 1000
+	maxOffsetThreshold                                      = 3000
+	defaultMaxProcessesTracked                              = 1024
+	defaultMaxTrackedConnections                            = 65536
+	defaultClosedConnectionsBufferThresholdRatioSystemProbe = 0.75
```

Comment:
Why is this 0.75? Don't we want this to be 1.0 to keep the status quo?

Reply:
removed this config entirely. also removed the atomic